.github/workflows/kotlin.yml

```diff
@@ -33,31 +33,23 @@ jobs :
           java-version : 11
 
       ## Actual task
-      - uses : burrunan/gradle-cache-action@v1
+      - uses: gradle/gradle-build-action@v2
         name : Assemble with gradle — make sure everything builds
         with :
-          gradle-dependencies-cache-key : |
-            gradle/libs.versions.toml
           arguments : |
-            assemble --no-daemon --stacktrace
-          concurrent : true
-          gradle-build-scan-report : false
-          gradle-distribution-sha-256-sum-warning : false
+            assemble --stacktrace
```

Comment:
I take it `--no-daemon` is built in now?

Reply:
More or less.
https://github.com/gradle/gradle-build-action#stopping-the-gradle-daemon
> By default, the action will stop all running Gradle daemons in the post-action step, prior to saving the Gradle User Home state. This allows for any Gradle User Home cleanup to occur, and avoid file-locking issues on Windows.
>
> If caching is unavailable or the cache is in read-only mode, the daemon will not be stopped and will continue running after the job is completed.